contribs/gmf/src/controllers/AbstractAppController.js

```diff
@@ -40,7 +40,7 @@ import gmfThemeThemes from 'gmf/theme/Themes.js';
  * @property {import("ol/style/Style.js").default} [positionFeatureStyle]
  * @property {import("ol/style/Style.js").default} [accuracyFeatureStyle]
  * @property {number} [geolocationZoom]
- * @property {string} [autorotate]
+ * @property {boolean|undefined} [autorotate]
```

Comment:
you normally don't need to add `undefined`: if the property name is optional (marked by using `[]`) typescript should accept `undefined`

Reply:
Oh, I didn't know that.  OK, I'll fix this.